Twilio/Twilio-CA-ESP-Sample/package-lock.json

```diff
@@ -1,13 +1,4864 @@
 {
```

Comment:
Should this package-lock file be excluded in the .gitignore?

Reply:
Since package-lock.json was already committed to the repo, I just kept things the way that they were already set.
I can remove package-lock.json and add it to .gitignore. 

Do you think that would be beneficial?

Reply:
don't we want them there for depandabot to update the project dependencies automatically?